LiteDB/Engine/Disk/Serializer/BufferPool.cs

```diff
@@ -35,5 +35,17 @@ public static void Return(byte[] buffer)
                 _bytePool.Return(buffer);
             }
         }
+
+        public static byte[] RentPageSizeBuff()
```

Comment:
If will return `byte[]` why not use direct `new byte[]` in code? This use is very limited and checked if are not inside a loop or heavy use

Reply:
I add this method, if in future improve BufferSlice. I wrote about this in comment:
https://github.com/mbdavid/LiteDB/pull/1258#issue-300993365